_applications/my-first-radanalytics-app.adoc

```diff
@@ -0,0 +1,113 @@
+= My First RADanalytics Application
+:page-link: my-first-radanalytics-app
+:page-weight: 0
+:page-labels: [Python, S2I]
+:page-layout: markdown
+:page-menu_template: menu_tutorial_application.html
+:page-menu_items:
+:page-description: In this tutorial you will learn how to create a source-to-image application for Apache Spark from the ground up. The source code is based on the upstream Pi calculator from the Apache Spark project examples with a slight twist, the addition of a web server to create an on-demand calculation microservice.
+
+[[introduction]]
+== Introduction
+
+In this tutorial you will build a microservice which will calculate an approximate value for https://en.wikipedia.org/wiki/Pi[Pi] when requested by HTTP. You will learn how to create applications which utilize Apache Spark and that are deployed directly from a source repository to OpenShift with a single command.
+
+This tutorial will walk you through every step to creating, deploying and operating an Apache Spark driver application on OpenShift. You should expect to spend 30 to 60 minutes on these exercises, depending on your familiarity.
+
+[[prerequisites]]
+== Prerequisites
+
+Before beginning the tutorial, there are a few prerequisites which are necessary to ensure you have the proper tools and resources to complete the work.
+
+* Access to an OpenShift cluster
+* A terminal with the `oc` command available
+* A new project in OpenShift to store your work
+* The Oshinko templates available in your project (see the link:/get-started[Get Started] page for instructions)
+* An editor to work on your source files
+* An online space for a Git repository to store your code, such as https://github.com[GitHub] or https://bitbucket.com[BitBucket]
+
+With these in place you are ready to begin creating your microservice.
+
+[[overview]]
+== Application overview
+
+The SparkPi microservice that you will build during this tutorial is quite simple in its design and functionality. SparkPi is deployed as a single server pod and several Apache Spark pods. The microservice provides an HTTP server that will accept `GET` requests and respond with an estimation of Pi, it does this by utilizing Apache Spark to calculate the value using the https://en.wikipedia.org/wiki/Approximations_of_%CF%80#Summing_a_circle.27s_area[summing of a circle's area] technique.
```

Comment:
another total grammar nit, I would either use a semicolon after "estimation of Pi" if you really want to or just add a period and start a new sentence with "It does this".

Reply:
done